docs/platform-engineers/system-operation/offline-installation.md

```diff
@@ -0,0 +1,60 @@
+# KubeVela Offline Installation Guide
+
+KubeVela offline installation includes the installation of KubeVela core and KubeVela Addon.
+
+## KubeVela Core offline installation
+
+### KubeVela chart
+
+- Download Helm Chart package of vela-core
```

Comment:
when subsection of the doc are logically organized in sequence, we suggest to use serial number `1,2,3..` instead of `-` for wrongly showing they're in parallel.

Reply:
This suggestion apply to all docs. Except for this, the rest is LGTM.